.github/workflows/package-submissions.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           $github = Invoke-RestMethod -uri "https://api.github.com/repos/Microsoft/PowerToys/releases" 
           
           $targetRelease = $github | Where-Object -Property name -match 'Release'| Select -First 1
-          $installerUrl = $targetRelease | Select -ExpandProperty assets -First 1 | Where-Object -Property name -match 'PowerToysSetup' | Select -ExpandProperty browser_download_url
+          $installerUrl = $targetRelease | Select -ExpandProperty assets -First 1 | Where-Object -Property name -match 'PowerToysSetup.*x64' | Select -ExpandProperty browser_download_url
```

Comment:
Could you elaborate this change?

Reply:
This is grabbing only x64. The script needs larger tweaks after a manual step to include arm. I created an issue to track.